app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
```

Comment:
Out of curiosity, why does this need to be defined in a separate file? It doesn't seem like something we'd especially need to be reusable.